geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/export/ExportCommand.scala

```diff
@@ -466,12 +467,13 @@ object ExportCommand extends LazyLogging {
       }
       exporter = new Exporter(options.copy(file = names.next), hints, queriedDictionaries)
       exporter.start(sft)
-      count = 0
+      count = 0L
     }
 
     @tailrec
     private def export(features: Iterator[SimpleFeature], result: Option[Long]): Option[Long] = {
-      val counter = features.take(estimator.estimate(exporter.bytes)).map { f => count += 1; f }
+      var estimate = estimator.estimate(exporter.bytes)
+      val counter = features.takeWhile { _ => count += 1; estimate -= 1; estimate >= 0 }
```

Comment:
Neat!  The export command can take a file size limit now?  

Is that documented?  What happens when the file overflows?

Reply:
yeah, that's been a feature for a while. it is documented, look for 'chunk-size'. it will spill over to a new file when it hits the limit.